go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/Azure/buffalo-azure
+
+require (
+	github.com/Azure/azure-sdk-for-go v18.0.0+incompatible
+	github.com/Azure/buffalo-azure/sdk v0.1.0
+	github.com/Azure/go-autorest v10.12.0+incompatible
+	github.com/cockroachdb/apd v1.1.0 // indirect
+	github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c // indirect
+	github.com/gobuffalo/buffalo v0.13.0
+	github.com/gobuffalo/buffalo-plugins v1.0.4
+	github.com/gobuffalo/fizz v1.0.12 // indirect
+	github.com/gobuffalo/makr v1.1.5
+	github.com/gobuffalo/pop v4.8.4+incompatible
+	github.com/jackc/fake v0.0.0-20150926172116-812a484cc733 // indirect
+	github.com/jackc/pgx v3.2.0+incompatible // indirect
+	github.com/jmoiron/sqlx v1.2.0 // indirect
+	github.com/joho/godotenv v1.3.0
+	github.com/markbates/inflect v1.0.1
+	github.com/marstr/collection v0.3.3 // indirect
+	github.com/marstr/randname v0.0.0-20180611202505-48a63b6052f1
+	github.com/mitchellh/go-homedir v1.0.0
+	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24 // indirect
+	github.com/sirupsen/logrus v1.1.1
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/viper v1.2.1
+)
+
+replace github.com/Azure/buffalo-azure/sdk => ./sdk
```

Comment:
Did you test what happens with this line if folks are cold-installing buffalo-azure using `go get`?

Reply:
I think we have a chicken before the egg problem here. I need to get a tagged module version into the repo before I can truly test this.

What I propose we do is to accept this PR, produce a tag for `sdk` and then issue a follow up PR to take a dependency on the tagged version of `sdk`.

Reply:
One thing to note, @marstr. I've added the dep for `./sdk` at v0.1.0. With the expectation that it would be tagged v0.1.0 for the initial tag. If that is not appropriate, let me know what version you'd prefer.